### ISSUE TITLE
Visual appearance of website (incl. search results)

### ISSUE THREAD

**Issue by Echolon** [bascht/omemo-top#79]
Hi,

if you search for this Website or get it as search result, it looks like this:

<img src="https://user-images.githubusercontent.com/29168011/30977556-271f15b2-a478-11e7-898b-4dd0892a74a4.PNG" height="83" width="533">

<img src="https://user-images.githubusercontent.com/29168011/30977555-271ec6a2-a478-11e7-8939-ea85d0228e6b.PNG" height="136" width="556">

For people like 'us' might be obvious what the ?OTR:.... means - for other, maybe normal users, this might looks like something awkward and/or they mistrust the website. I think we should erase this. Furthermore, the website issues OMEMO and not OTR anymore (even so, OTR is part of OMEMO).

What do you think?

Cheers

**Comment by bascht** [comment]
@Echolon You're right. I didn't notice the `<meta` tag that Jekyll was adding to the markup. I've polished it up a bit in #81. Can you take a look?

**Comment by Echolon** [comment]
Done, seems to be okay

**Comment by bascht** [comment]
@Echolon close #79 at will. :)

**Comment by bascht** [comment]
Yip, it took a while. It's updated now:

![screenshot_20171012_153418](https://user-images.githubusercontent.com/51264/31498660-dee92d1e-af62-11e7-87e2-4237e5c06c99.png)

Come to think of it, the description line could be even longer. 😄

**Comment by Echolon** [comment]
Wonder why Google for years never though of useing the free space on wide-screen.... It's annoying...

But good to see it's now more clear for everyone
